interface/views.py

```diff
@@ -153,10 +153,11 @@ def done(request, pk):
         log.warning('Score is None')
```

Comment:
To test this part of the code:
* comment out `assert submission.verify_jwt(request.GET.get('token'))`
* Change `stdout` and `stderr` to
```
stdout = options['stdout']
stderr = options['stderr']
```
* have at least one submission in the database

Make a POST request(maybe `curl`) to `http://localhost:8100/submission/1/done` with the following JSON:
```
{
"stdout": "what stdout you want! Must have TOTAL: 170/170",
"stderr": "error maybe",
"exit_code": 0
}
```
Now you can check in the `admin` tab, under `submissions` if the submission with the id 1 has the correct `stdout` and `stderr` values.